Internals/Events/message/GAB.MessageCreate.js

```diff
@@ -326,39 +327,57 @@ class MessageCreate extends BaseEvent {
-										};
-										const commandData = {
-											name: this.client.getPublicCommandName(msg.command),
-											usage: this.client.getPublicCommandMetadata(cmd).usage,
-											description: this.client.getPublicCommandMetadata(cmd).description,
-										};
-										await this.client.getPublicCommand(cmd)(botObject, documents, msg, commandData);
-									} catch (err) {
-										winston.warn(`Failed to process command "${cmd}"`, { svrid: msg.guild.id, chid: msg.channel.id, usrid: msg.author.id }, err);
-										this.client.logMessage(serverDocument, LoggingLevels.ERROR, `Failed to process command "${cmd}" X.X`, msg.channel.id, msg.author.id);
+									const commandFunction = this.client.getPublicCommand(cmd);
+									if (!commandFunction) {
+										const commandList = this.client.getPublicCommandList()
+											.filter(c => this.client.getPublicCommand(c))
+											.join(", ");
 										msg.send({
 											embed: {
-												color: Colors.ERROR,
-												title: `Something went wrong! 😱`,
-												description: `Something went wrong while executing \`${cmd}\`!\n**Error Message**: \`\`\`js\n${err.stack}\`\`\``,
+												color: Colors.SOFT_ERR,
+												title: `Hold on! ✋`,
+												description: `The command \`${cmd}\` is not implemented yet!\nThis version of GAwesomeBot is still in heavy development and many commands do not work yet. 🚧\nHere are all the commands that are supported right now:\n\`\`\`${commandList}\`\`\``,
 												footer: {
-													text: `Contact your GAB maintainer for more support.`,
+													text: `Over time more and more commands will be added. Contact your GAB maintainer for more support.`,
 												},
 											},
 										});
+									} else {
+										try {
+											const botObject = {
+												client: this.client,
+												configJS: this.configJS,
+												utils: Utils,
+												Utils,
+												Constants,
+											};
+											const documents = {
+												serverDocument,
+												channelDocument,
+												memberDocument,
+												userDocument,
+											};
+											const commandData = {
+												name: cmd,
+												usage: metadata.usage,
+												description: metadata.description,
+											};
+											await commandFunction(botObject, documents, msg, commandData);
+										} catch (err) {
+											winston.warn(`Failed to process command "${cmd}"`, { svrid: msg.guild.id, chid: msg.channel.id, usrid: msg.author.id }, err);
+											this.client.logMessage(serverDocument, LoggingLevels.ERROR, `Failed to process command "${cmd}" X.X`, msg.channel.id, msg.author.id);
+											const description = Math.random() > 0.001 ? `Something went wrong while executing \`${cmd}\`!\n**Error Message**: \`\`\`js\n${err.stack}\`\`\`` : "OOPSIE WOOPSIE!! Uwu We made a fucky wucky!! A wittle fucko boingo! The code monkeys at our headquarters are working VEWY HAWD to fix this!";

```

Comment:
No.

Reply:
Should I add a few more zeros? ^^

Reply:
If require("gar")(process.argv.slice(2)) has "owo" set to true you should display that message instead of the stack.

Reply:
That way, we can start GAB in --owo mode